src/types.ts

```diff
@@ -255,4 +255,18 @@ export interface AppManifest {
   Be aware that subscription queries are required in manifest sections
   */
   webhooks?: WebhookManifest[];
+  /**
+   * Allows app installation for specific Saleor versions, using semver.
+   * https://www.npmjs.com/package/semver
```

Comment:
```suggestion
   * https://github.com/npm/node-semver#versions
```
Maybe a link directly to this section can be more informative?